### DISCUSSION TITLE
seqr on AWS

### DISCUSSION THREAD

**Discussion by equinne5** [broadinstitute/seqr#3313] [General]
Hi, apologies if this has been asked before I was just wondering if its possible to run a local install of seqr on AWS - is it possible to have a copy of the reference files in storage and somehow reference them when running `docker-compose exec pipeline-runner load_data.sh`? 
thanks!

**Comment by hanars** [comment]
So in theory seqr is platform agnostic so you should be able to get a basic AWS seqr instance by running the instructions for local setup on a single VM in AWS. You would be responsible for either running `download_reference_data.sh` in AWS yourself or running it locally and then copying those files up to AWS. We only provide explicit support for GCP's dataproc, so you would need to run the loading using the "local" load_data script, theres no way to use any of AWS task runners at this time.

Also, any of the helper components specified in the docker compose file, such as redis, postgres, and elasticsearch, can be replaced with AWS services and then updating the seqr env's `<SERVICE>_SERVICE_HOSTNAME` variables. But we do not have any real documentation for how this would work, you would be responsible for figuring out how to get those services configured in AWS. 

I believe @monkollek might have gotten seqr working on AWS at some point, but I don't know if he has any documentation/forks that would be helpful

**Reply by equinne5** [reply]
really appreciate the response- thanks for your help!
